permissions/plugin-ca-apm.yml

```diff
@@ -3,6 +3,12 @@ name: "ca-apm"
 github: "jenkinsci/ca-apm-plugin"
 paths:
 - "org/jenkins-ci/plugins/ca-apm"
+- "org/jenkins-ci/plugins/ca-apm/ca-apm-api"
+- "org/jenkins-ci/plugins/ca-apm/ca-apm-core"
+- "org/jenkins-ci/plugins/ca-apm/ca-apm-default-strategy"
+- "org/jenkins-ci/plugins/ca-apm/ca-apm-dist"
+- "org/jenkins-ci/plugins/ca-apm/ca-apm-plugin"
```

Comment:
Can this rather be suffixed `-parent`. That is the common practice, and I envision a slight confusion for people with a `-plugin` suffix (which we are generally trying to avoid for plugins, I understand for your case it's the top-level parent pom of your multimodule maven project)

Reply:
the project structure as:
ca-apm-plugin(root pom)
-- ca-apm-api (jar)
-- ca-apm-core (jar)
-- ca-apm-default-strategy (jar)
-- ca-apm (hpi)
-- ca-apm-dist (pom)

Yes we can rename root project to "ca-apm-parent"
I put the name ca-apm-plugin due to repository name.